runtime/compiler/makefile.ftl

```diff
@@ -87,6 +87,9 @@ export BUILD_CONFIG?=prod
 <#elseif uma.spec.id?starts_with("linux_arm_linaro")>
   export PLATFORM=arm-linux-gcc-cross
 </#if>
+<#if uma.spec.id?starts_with("osx_x86-64")>
```

Comment:
Maybe try something like
```
<#if uma.buildinfo.version.major == 8>
   export PLATFORM=amd64-osx-gcc
<#else>
   export PLATFORM=amd64-osx-clang
</#if>
```

Reply:
Rebased and updated to handle v8 difference.